src/js/components/FileInput/doc.js

```diff
@@ -42,18 +42,17 @@ export const doc = FileInput => {
       PropTypes.bool,
       PropTypes.shape({
         aggregateThreshold: PropTypes.number,
+        max: PropTypes.number,
       }),
     ])
       .description(
         `Whether to allow multiple files. If an object is supplied,
         'aggregageThreshold' indicates the maximum number of individual
         files to show. Above this, only a single message describing the
-        number of files will be shown.`,
+        number of files will be shown.And if the object 'max' is supplied.
```

Comment:
wording suggestion to align with aggregateThreshold:

If an object is supplied, 'max' indicates the maximum number of files that can be added. If the user has reached the max, no additional files will be able to be added.

Reply:
Done